### PR TITLE
Passive Gate Icon Fix And Naming

### PR DESCRIPTION
**Pull request by Arvenius169**
## What Does This PR Do
Swaps the icons of East and West oriented passive gate pipe fittings so that they appear(correctly) to be oriented the same direction as East and West passive gates(wrenched).

Also allows passive gates to be named with pens, exactly like pumps.

## Why It's Good For The Game
Because it's currently wrong, and passive gates appear to change from East to West and vice versa when wrenched down or wrenched up.

Also sometimes you just wanna name a passive gate.

## Images of changes
PG

## Changelog
:cl:
fix: Corrected icons of passive gate pipe fittings.
tweak: Lets you name passive gates with a pen.
/:cl:

### CODE REVIEW

**Comment by Citinited** [comment]
lol I always assumed that the flipping was just a feature

good change

File: code/ATMOSPHERICS/components/binary_devices/passive_gate.dm [hunk 1]
```diff
@@ -184,7 +184,15 @@
 	SSnanoui.update_uis(src)
 
 /obj/machinery/atmospherics/binary/passive_gate/attackby(obj/item/W, mob/user, params)
-	if(!istype(W, /obj/item/wrench))
+	if(istype(W, /obj/item/pen))
+		var/t = copytext(stripped_input(user, "Enter the name for the gate.", "Rename", name), 1, MAX_NAME_LEN)
```
Comment (farie82) [inline code/ATMOSPHERICS/components/binary_devices/passive_gate.dm:188]: `copytext` is not required here due to `stripped_input` already capping the character limit

File: code/ATMOSPHERICS/components/binary_devices/passive_gate.dm [hunk 2]
```diff
@@ -184,7 +184,15 @@
...
-	if(!istype(W, /obj/item/wrench))
+	if(istype(W, /obj/item/pen))
+		var/t = copytext(stripped_input(user, "Enter the name for the gate.", "Rename", name), 1, MAX_NAME_LEN)
+		if(!t)
+			return
+		if(!in_range(src, usr) && loc != usr)
+			return
+		name = t
+		return
+	else if(!istype(W, /obj/item/wrench))
```
Comment (farie82) [inline code/ATMOSPHERICS/components/binary_devices/passive_gate.dm:195]: needs to change due to the tool refactor.

**Comment by Fox-McCloud** [comment]
Please deconflict. Otherwise, quite stale.

**Comment by Fox-McCloud** [comment]
Failure to respond. Still has conflicts. Stale.

**Pull request closed without merging**